tests/drivers/gpio/gpio_basic_api/frdm_k64f.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	resources {
+		compatible = "test,resources";
+		gpios =
+			<&gpioc 16 0>, <&gpioc 17 0>,
```

Comment:
Just curious how this set or why its this set of GPIOs?

Reply:
I needed two on the same port, hence C16 and C17 (also those are Arduino D0 and D1, which made some sense).

For the performance test that isn't in this PR I needed more on the same port, so added the others.  They can be taken out.